staging/src/k8s.io/apiextensions-apiserver/examples/client-go/main.go

```diff
@@ -38,11 +37,12 @@ import (
 )
 
 func main() {
+	masterUrl := flag.String("master", "", "The address of the Kubernetes API server. Overrides any value in kubeconfig. Only required if out-of-cluster.")
```

Comment:
`masterUrl` -> `masterURL`. To make golint happy.

Reply:
Done. :)